### ISSUE TITLE
Increase maximum allowed `Font Size`

### ISSUE THREAD

**Issue by soof-golan** [warpdotdev/Warp#4395]
### Describe the solution you'd like?

Please remove the upper limit of the _Font Size_ setting of `25` (_pts_? _pixels_?)

## Problem:

I'm using Warp as my daily driver and also for my video streams, I'm using a 27 inch 1440p monitor.
The maximum allowed font size of 25 is too small to be legible for mobile stream viewers.

## My current workaround

Opening the settings menu and manually inputting 40.

Problem with work around: Any CMD + ➕   or CMD + ➖ reset the font back to 25

## Video demo

https://github.com/warpdotdev/Warp/assets/83900570/60f04c39-24b4-4ab2-a226-b7ece51a6dce

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by soof-golan** [comment]
btw, would have loved to contribute this fix if your terminal was open source.

**Comment by dannyneira** [comment]
You're right about the font size limitation in Warp. Warp currently has a maximum font size limit of 25 points/pixels when using the CMD+Plus shortcut (or Ctrl+Plus on Windows/Linux). However, there is a workaround:

You can manually set a larger font size by:
1. Opening Warp
2. Going to Settings > Appearance > Font
3. Manually inputting your desired font size (users have reported being able to set it up to 100)

To anyone else facing this issue, please add a 👍 to the original post at the top or comment with your details, and subscribe if you'd like to be notified.
